### DISCUSSION TITLE
Probleme pour initialiser l'interface

### DISCUSSION THREAD

**Discussion by theflyingfish-afk** [pretrehr/Sports-betting#32] [General]
Bonjour,
Tout d'abord désolé d'avance si mes questions sont stupides mais je suis un véritable newbie en python et en programmation.
Quand j'essaye de lancer l'interface avec la commande "python interface_pysimplegui.py" j'ai un message d'erreur : 
<img width="565" alt="Capture d’écran 2021-03-28 à 00 30 45" src="https://user-images.githubusercontent.com/81489969/112737929-b1588b80-8f5e-11eb-9e5a-0c379c1567a1.png">
J'ai donc rajouté la ligne de code  # -*- coding: utf-8 -*- (merci google)
Cependant nouvelle erreur :
<img width="610" alt="Capture d’écran 2021-03-27 à 23 51 02" src="https://user-images.githubusercontent.com/81489969/112738008-73a83280-8f5f-11eb-97fd-2c15542373a9.png">

Je ne pense pas qu'il y ai une erreur du code puisque je n'y ai pas touché, auriez-vous donc une idée de ce qui ne fonctionne pas dans mon installation ? (le fichier "requierment.txt" s'est installé avec succès)

Amicalement,
the flying fish

**Comment by pretrehr** [comment]
Bonjour,
Dans tous les cas, il faut exécuter le fichier avec Python en version >= 3.6 donc si vous faîtes `python --version` et que cela indique un numéro de version commençant par 2 (par exemple 2.7.1), alors ce n'est pas la bonne version de Python
Selon le système d'exploitation que vous avez, il est possible que vous deviez appeler `python3 interface_pysimplegui.py` à la place de `python interface_pysimplegui.py`, cela devrait résoudre votre problème.
N'hésitez pas à revenir vers moi si cela ne fonctionne toujours pas.

**Reply by theflyingfish-afk** [reply]
Merci pour votre réponse ! En appelant python3 ça fonctionne effectivement :)
